smoketests/helpers.js

```diff
@@ -17,7 +21,121 @@ const swapPkgName = (current, isSubPackage = false) => {
     fs.renameSync(getPkgPath(current, isSubPackage), getPkgPath(next, isSubPackage));
 };
 
+const CLI_ENTRY_PATH = getBinPath();
```

Comment:
```suggestion
const CLI_ENTRY_PATH = path.resolve(ROOT_PATH, './packages/webpack-cli/bin/cli.js');;
```

`getBinPath()` isn't used anywhere else.

Reply:
Done 👍🏻

Reply:
Same as the case with `swapPkgName()`.

Reply:
`swapPkgName()` is used multiple times in this file.

Reply:
I mean, It need not be exported.

Reply:
updated 👍🏻